### PR TITLE
start 2.8.0 rc1

### PR DESCRIPTION
**Pull request by garyverhaegen-da**

### CODE REVIEW

File: LATEST [hunk 1]
```diff
@@ -1,4 +1,4 @@
-a821d52fe12af990aa8e73a3767f9010cbe83bd3 2.8.0-snapshot.20231116.12377.0.va821d52f SPLIT_RELEASE
+45e730b92af78d35f065d3031b31f064572036c2 2.8.0-snapshot.20231122.12398.0.v45e730b9 SPLIT_RELEASE
```
Comment (rgugliel-da) [inline LATEST:1]: Is it on purpose that you did not include https://github.com/digital-asset/daml/commit/81e3ed1d9da4a0a4f5f1c020596f0554316037f5 ? Does that make any difference ?

**Review by rgugliel-da — APPROVED** [review]
LGTM
Left a question

File: LATEST (hunk 1 above)
Comment (garyverhaegen-da) [reply LATEST:1]: That was merged after the release branch was created. I don't _think_ it matters as we replace canton with the appropriate version when we repackage the SDK in assembly anyway.

@remyhaemmerle-da @basvangijzel-DA Any opinion on this? Does https://github.com/digital-asset/daml/pull/17881 need to be backported to the release branch?

**Review by remyhaemmerle-da — APPROVED** [review]

**Comment by garyverhaegen-da** [comment]
Bumped up one commit to include https://github.com/digital-asset/daml/pull/17899.

**Review by remyhaemmerle-da — APPROVED** [review]

**Comment by garyverhaegen-da** [comment]
Updated to include https://github.com/digital-asset/daml/pull/17902.

**Comment by garyverhaegen-da** [comment]
Bump to include https://github.com/digital-asset/daml/pull/17909.

**Pull request merged**